### ISSUE TITLE
linkColour makes edges invisible unless a single colour or a JS function

### ISSUE THREAD

**Issue by martinjhnhadley** [christophergandrud/networkD3#102]
Setup code

``` r
library(networkD3)
data(MisLinks)
data(MisNodes)
foo_links <- MisLinks[1:10,]
foo_nodes <- MisNodes[1:10,]
```

The following will display a graph with edges coloured red:

``` r
forceNetwork(Links = foo_links, Nodes = foo_nodes, Source = "source",
             Target = "target", Value = "value", NodeID = "name",
             Group = "group", opacity = 0.4, zoom = TRUE,
             linkColour = "red",
             linkWidth = 2
)
```

Providing a vector to linkColour results in the edges becoming invisible, even if only one colour is used

``` r
colourVector <- rep("red",10)
forceNetwork(Links = foo_links, Nodes = foo_nodes, Source = "source",
             Target = "target", Value = "value", NodeID = "name",
             Group = "group", opacity = 0.4, zoom = TRUE,
             linkColour = colourVector,
             linkWidth = 2
             )
```

As offered on SE (http://stackoverflow.com/a/34482817/1659890) providing a JS function works:

``` r
forceNetwork(Links = foo_links, Nodes = foo_nodes,
             Source = "source", Target = "target",
             Value = "value", NodeID = "name",
             Group = "group", opacity = 0.8,
             linkColour = JS('function(l) { return l.value > 1 ? "red" : "blue" }'))
```

**Comment by cjyetman** [comment]
Can you please try this again with the currently released version (0.3)? The following seems to work for me...
```r
library(networkD3)
data(MisLinks)
data(MisNodes)
foo_links <- MisLinks[1:10,]
foo_nodes <- MisNodes[1:10,]

colourVector <- c(rep("red",5),rep("blue",5))

forceNetwork(Links = foo_links, Nodes = foo_nodes, Source = "source",
             Target = "target", Value = "value", NodeID = "name",
             Group = "group", opacity = 1, linkColour = colourVector)
```

**Comment by martinjhnhadley** [comment]
@cjyetman thanks! This now works for me too 👍 
Assume it's decorum to close, and so I have.
